partner_risk_insurance/migrations/15.0.3.0.1/pre-migration.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2024 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    """Create account move columns to performance upgrade on large databases"""
+    openupgrade.logged_query(
+        env.cr,
+        """
+    ALTER TABLE account_move
+        ADD COLUMN IF NOT EXISTS insured_with_credit_policy boolean DEFAULT false,
+        ADD COLUMN IF NOT EXISTS credit_policy_company_id INT,
+        ADD COLUMN IF NOT EXISTS credit_policy_state_id INT;
+    """,
+    )
```

Comment:
You don't need to repeat code:

```suggestion
from odoo.addons.partner_risk_insurance.hooks import pre_init_hook


def migrate(cr, version):
    """Create account move columns to performance upgrade on large databases"""
    pre_init_hook(cr)
```